### DISCUSSION TITLE
Changing Actuation Point?

### DISCUSSION THREAD

**Discussion by shabbyfox** [kanru/hhkb-zmk#18] [Q&A]
Hello,

How to change the actuation point in the firmware? Most of the keys of my board actuate before the bump. :(
I've looked in zmk,kscan-hhkb-pro2.yaml and not sure which one is for actuation level.

🙏🙏

**Comment by kanru** [comment]
> Hello,
> 
> How to change the actuation point in the firmware? Most of the keys of my board actuate before the bump. :(
> I've looked in zmk,kscan-hhkb-pro2.yaml and not sure which one is for actuation level.
> 
> 🙏🙏

Unfortunately the actuation calibration data is fixed in the chip on the matrix board. We only read 1 or 0 from the ADC line.

You may try to change the keyscan logic to require N positive reading to send the key down signal to add a delay to the actuation.
